scripts/apps/search/services/SearchService.js

```diff
@@ -337,12 +337,24 @@ export function SearchService($location, gettext, config, session) {
                 criteria.post_filter = {'and': post_filters};
             }
 
-            if (search.q) {
+            var query_string = null;
+            if (search.q && !search.raw) {
+                query_string = search.q.replace(/\//g, '\\/');
+            } else {
```

Comment:
this part might be a bit more readable as functional:
```
query_string = [search.q, search.raw].filter(q => q).map(q => '(' + q.replace(...) + ')').join(' AND ');
```
but maybe it's not :wink: